pyproject.toml

```diff
@@ -30,7 +30,7 @@ classifiers=[
 ]
 dependencies = [
     "torch",
-    "qadence[braket, pulser, horqrux]",
+    "qadence[braket, pulser, horqrux]<1.8.0",
```

Comment:
I guess you can remove the braket dependency ? I wouldn't set an upper bound for the versions though.

Reply:
Ok, let me fix it to 1.7.8 and remove pulser then